src/main/java/duke/parser/Parser.java

```diff
@@ -47,6 +53,13 @@ public static CommandEnum valueOfLabel(String label) {
         }
     }
 
+    /**
+     * Parses user input into an executable command.
+     *
+     * @param fullCommand text to be parsed into an executable command
+     * @return a Command for Duke program to execute
+     * @throws DukeException if the command executed has errors
+     */
     public static Command parse(String fullCommand) throws DukeException {
         String[] tokens = fullCommand.split("\\s+", 2);
```

Comment:
tokens can be confusing, maybe try words? (wasn't intuitive to me what was the variable for until i saw the code however i feel that it is just a tiny change you might want to consider!)